### PR TITLE
Create docker-compose.yml

### PR DESCRIPTION
**Pull request by whitlocktech**
A proposed docker compose for your image. I used it to setup my controller with your image.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (mbentley) [inline docker-compose.yml:1]: Nit: extra space isn't needed

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+version: "3.1"
+
+services:
+   omada-controller:
+     container_name: Omada
```
Comment (mbentley) [inline docker-compose.yml:6]: change `Omada` to `omada-controller`

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+     image: mbentley/omada-controller:4.2
+     environment:
+      - TZ=America/Chicago
```
Comment (mbentley) [inline docker-compose.yml:9]: Change to `TZ=Etc/UTC`

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      - MANAGE_HTTP_PORT=8088
+      - MANAGE_HTTPS_PORT=8043
+      - PORTAL_HTTP_PORT=8088
+      - PORTAL_HTTPS_PORT=8043
+      - SHOW_SERVER_LOGS=true
+      - SHOW_MONGODB_LOGS=false
+#      - SSL_CERT_NAME="tls.crt"
```
Comment (mbentley) [inline docker-compose.yml:16]: Remove comment - it's the default certs.

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+#      - SSL_KEY_NAME="tls.key"
```
Comment (mbentley) [inline docker-compose.yml:17]: Remove comment - it's the default certs.

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+     network_mode:host
+     ports:
```
Comment (mbentley) [inline docker-compose.yml:19]: Remove the entire ports section for `network_mode: host` ignores ports.

File: docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      - TZ=America/Chicago
+      - MANAGE_HTTP_PORT=8088
+      - MANAGE_HTTPS_PORT=8043
+      - PORTAL_HTTP_PORT=8088
+      - PORTAL_HTTPS_PORT=8043
+      - SHOW_SERVER_LOGS=true
+      - SHOW_MONGODB_LOGS=false
+#      - SSL_CERT_NAME="tls.crt"
+#      - SSL_KEY_NAME="tls.key"
+     network_mode:host
```
Comment (mbentley) [inline docker-compose.yml:18]: Nit: space between the colon and `host`.

File: docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+      - 8843:8843
+      - 29810:29810
+      - 29810:29810/udp
+      - 29811:29811
+      - 29811:29811/udp
+      - 29812:29812
+      - 29812:29812/udp
+      - 29813:29813
+      - 29813:29813/udp
+     volumes:
```
Comment (mbentley) [inline docker-compose.yml:31]: For each of the volumes, remove the `<Insert Path here>/`

File: docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+      - <Insert Path here>/omada-data:/opt/tplink/EAPController/data
+      - <Insert Path here>/omada-work:/opt/tplink/EAPController/work
+      - <Insert Path here>/omada-logs:/opt/tplink/EAPController/logs
+     restart: always
```
Comment (mbentley) [inline docker-compose.yml:35]: Change to `unless-stopped`

**Review by mbentley — CHANGES_REQUESTED** [review]
Thanks for the PR!  Typically when I have an example or a Docker Compose file (in other projects), I try to keep all of the settings to defaults for the image, which is why all of the comments.

Thanks!

**Comment by whitlocktech** [comment]
I will update it even I get home this week. Port section was leftover from before I used ghost mode. But I will update it based on your comments.

**Comment by whitlocktech** [comment]
@mbentley All changes have been made per request

**Review by mbentley — APPROVED** [review]
Thanks for the updates.  LGTM

**Pull request merged**